src/core/Akka.Cluster.Tests.MultiNode/JoinWithOfflineSeedNodeSpec.cs

```diff
@@ -0,0 +1,196 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="JoinWithOfflineSeedNodeSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Immutable;
+using Akka.Actor;
+using Akka.Actor.Dsl;
+using Akka.Cluster.TestKit;
+using Akka.Event;
+using Akka.Remote.TestKit;
+using FluentAssertions;
+
+namespace Akka.Cluster.Tests.MultiNode
+{
+    public class JoinWithOfflineSeedNodeConfig : MultiNodeConfig
+    {
+        public RoleName Seed { get; }
+
+        public RoleName NonSeed { get; }
+
+        public JoinWithOfflineSeedNodeConfig()
+        {
+            Seed = Role("seed");
+            NonSeed = Role("nonseed");
+
+            CommonConfig = DebugConfig(false).WithFallback(MultiNodeClusterSpec.ClusterConfig());
+        }
+    }
+
+    public class JoinWithOfflineSeedNodeSpecNode1 : JoinWithOfflineSeedNodeSpec { }
+    public class JoinWithOfflineSeedNodeSpecNode2 : JoinWithOfflineSeedNodeSpec { }
+
+    /// <summary>
+    /// Tests to ensure that if we have 2 seed nodes defined and 1 is offline during a <see cref="Cluster.JoinSeedNodes"/>
+    /// command, the joining node will still be marked as <see cref="MemberStatus.Up"/> as long as there are no unreachable nodes
+    /// within the cluster.
+    /// </summary>
```

Comment:
Comment explains the purpose of this spec - tests a common scenario where only 1 seed node needs to be Up in order for a cluster to be joined. Other previous seed nodes can be Downed / offline and the join will still go through correctly.